src/umpire/util/backtrace.hpp

```diff
@@ -0,0 +1,151 @@
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
Shouldn't this be named backtrace.inl?

Reply:
Potentially. I'm not sure whether there is that much benefit to maintaining inl file for something so small.

Reply:
Just trying to be consistent since we have other .inl files in Umpire.